src/js/profile/wearable/widget/wearable/SnapListview.js

```diff
@@ -379,7 +376,9 @@
 					removeSelectedClass(self);
 				}
 
-				self._listItemAnimate(event && event.detail && event.detail.scrollTop);
+				if (event && event.detail) {
```

Comment:
how about 
if (event && event.detail && event.detail.scrollTop) {
	self._listItemAnimate(event.detail.scrollTop);
}

Reply:
0 is correct values for scroll (scrollTop). We should pass it to _listItemAnimate.

Reply:
OK